src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,66 @@
+package mate.academy.model;
+
+import java.time.LocalDateTime;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "movie_session")
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @OneToOne
```

Comment:
Can we have more than one movieSession for one movie?

Reply:
Ok, maybe I don't understand the conditions. Changed to ManyToOne